types/relay-runtime/lib/store/RelayModernQueryExecutor.d.ts

```diff
@@ -20,7 +20,7 @@ export interface ExecuteConfig {
     readonly operation: OperationDescriptor;
     readonly operationExecutions: Map<string, ActiveState>;
     readonly operationLoader: OperationLoader | null | undefined;
-    readonly operationTracker?: OperationTracker | null | undefined;
+    readonly operationTracker?: OperationTracker | null;
```

Comment:
I don't understand why `undefined` was removed, though I see the `?` covers it as an option

Reply:
OK, I can see you're just being consistent now.  Maybe even a linter picked this up.  EIther was :+1: